apple/swift-docc-render
# Don't apply "smart indent" to parent symbols

Bug/issue #, if applicable: 86438798

## Summary

This fixes a minor issue where the "smart indent" logic/styling is mistakenly being applied to declarations for Objective-C classes/structs/protocols/enums when it shouldn't be.

Here is an example of how an Objective-C class declaration is currently being rendered due to this bug:
![bad rendering example](https://user-images.githubusercontent.com/212918/148464824-5537b826-a0b2-4c34-8350-65379548c1c1.png)

Here is how it should look with the changes from this branch:
![good rendering example](https://user-images.githubusercontent.com/212918/148464837-6b15ef49-ac6f-4526-93c7-a47f6725669b.png)

## Testing

Steps:
1. Download/unzip this [TestAtObjC.doccarchive.zip](https://github.com/apple/swift-docc-render/files/7825231/TestAtObjC.doccarchive.zip) test fixture
2. Configure `VUE_APP_DEV_SERVER_PROXY=/path/to/TestAtObjC.doccarchive` and run `npm run serve`
3. Open http://localhost:8080/documentation/testatobjc/mytestclass?language=objc in the browser
4. Verify that the declaration is on a single line now

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
